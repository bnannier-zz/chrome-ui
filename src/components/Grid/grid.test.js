import React from "react";
import { configure, mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

import { Grid } from "../";

describe("Grid Tests", () => {
  it("Default", function() {
    const wrapper = mount(<Grid>test</Grid>);
    expect(wrapper).toMatchSnapshot();
  });
});
