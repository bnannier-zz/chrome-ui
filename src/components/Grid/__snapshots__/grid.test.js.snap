// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid Tests Default 1`] = `
<Grid>
  <Base>
    <div
      className="mdc-layout-grid"
    >
      <GridInner>
        <Base>
          <div
            className="mdc-layout-grid__inner"
          >
            test
          </div>
        </Base>
      </GridInner>
    </div>
  </Base>
</Grid>
`;
